chat-android/src/test/java/com/ably/chat/BuildConfigTest.kt

```diff
@@ -0,0 +1,12 @@
+package com.ably.chat
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class BuildConfigTest {
+
+    @Test
+    fun `should hold app version without extra quotes`() {
+        assertEquals("0.1.0", BuildConfig.APP_VERSION)
```

Comment:
Could do this with a RexExp to avoid having to update the test on every release, but not too fussed here